tsconfig.json

```diff
@@ -15,7 +15,8 @@
   
       /* Emit */
       "sourceMap": true, /* Create source map files for emitted JavaScript files. */
-      "outDir": "./dist", /* Specify an output folder for all emitted files. */
+      "outDir": "./lib", /* Specify an output folder for all emitted files. */
```

Comment:
If you change the output folder, we need to update the `Makefile`, `.prettierignore`, `jest.config.js` and .`gitignore` file